tools/tf2openapi/types/constants.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+const (
+	servingMetaGraphTag    string = "serve"
```

Comment:
Your linter should be complaining about these. 

ConstsShouldBeCapitolCamelCase